src/lib/db/user-store.ts

```diff
@@ -111,31 +113,33 @@ class UserStore implements IUserStore {
         throw new Error('Can only find users with id, username or email.');
     }
 
+    buildBaseQuery(): any {
```

Comment:
I'd rename it to what it does. BaseQuery is not very meaningful. Sth like activeUsers()